docs/faq.md

```diff
@@ -256,3 +256,17 @@ Merged objects will always contain all samples in the given project ([see which
 
 If you would like to work with a merged object that only contains a subset of project samples, we recommend downloading the merged object and subsetting it directly to the samples of your choosing.
 See {ref}`Subsetting the Merged Object<getting_started:Subsetting the merged object>` for instructions on how to subset `SingleCellExperiment` and `AnnData` merged objects.
+
+
+## Why did project options change when I appended samples to My Dataset?
+
+If you would like to include samples from a dataset you have previously created, you can append these samples to your current `My Dataset`.
```

Comment:
```suggestion
If you would like to include all samples from a dataset you have previously created, you can append these samples to your current `My Dataset`.
```

Tweaked it to imply that all of the samples in the previous dataset would get moved and they would not get to pick and choose.